### DISCUSSION TITLE
Regular expression for TTS Text Replacement

### DISCUSSION THREAD

**Discussion by ngkwtys** [pilot51/voicenotify#101] [Q&A] [answered]
Hi, can I use regular expressions in TTS Text Replacement?
If this is possible, it would allow me to quickly find out the expected seismic intensity of an earthquake.
Unfortunately, most earthquake warning apps only scare people with noisy alerts, but Voice Notify has the potential to calmly respond to earthquakes.

**Comment by pilot51** [comment, marked as answer]
Sorry, not at the moment. It's a good idea. I created issue #102 from this discussion and hopefully one day I'll get around to implementing it.
